diff --git a/CMakeLists.txt b/CMakeLists.txt
index 31cb4c0ff..0d949804a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -374,7 +374,7 @@ set(MINIMUM_BOOST_VERSION "1.66.0")
 set(_boost_components "system;filesystem;thread;log;locale;regex;chrono;atomic;date_time;iostreams;nowide")
 find_package(Boost ${MINIMUM_BOOST_VERSION} REQUIRED COMPONENTS ${_boost_components})
 
-find_package(Eigen3 3.3.7 REQUIRED)
+find_package(Eigen3 REQUIRED NO_MODULE)
 
 add_library(boost_libs INTERFACE)
 add_library(boost_headeronly INTERFACE)
