diff --git a/Makefile b/Makefile
index 5783e660..b2cad98c 100644
--- a/Makefile
+++ b/Makefile
@@ -6,7 +6,6 @@
 AOMP ?= $(HOME)/rocm/aomp
 AOMP_REPOS = $(shell pwd)
 all: 
-	AOMP=$(AOMP) AOMP_REPOS=$(AOMP_REPOS) AOMP_APPLY_ROCM_PATCHES=0 TARBALL_INSTALL=1 $(AOMP_REPOS)/aomp/bin/build_prereq.sh
 	AOMP=$(AOMP) AOMP_REPOS=$(AOMP_REPOS) AOMP_APPLY_ROCM_PATCHES=0 TARBALL_INSTALL=1 DISABLE_LLVM_TESTS=1 $(AOMP_REPOS)/aomp/bin/build_aomp.sh
 install:
 	@echo "Installation complete to $(AOMP)"
diff --git a/aomp/bin/build_aomp.sh b/aomp/bin/build_aomp.sh
index f3283c11..a362f120 100755
--- a/aomp/bin/build_aomp.sh
+++ b/aomp/bin/build_aomp.sh
@@ -81,13 +81,12 @@ date
 echo " =================  START build_aomp.sh ==================="   
 echo 
 if [ "$AOMP_STANDALONE_BUILD" == 1 ] ; then
-  components="prereq rocm-cmake project roct rocr libdevice openmp extras comgr rocminfo"
+  components="rocm-cmake project roct rocr libdevice comgr rocminfo"
   _hostarch=`uname -m`
   # The rocclr architecture is very x86 centric so it will not build on ppc64. Without
   # rocclr, we have no HIP or OpenCL for ppc64 :-( However, rocr works for ppc64 so AOMP works.
   if [ "$_hostarch" == "x86_64" ] ; then
     # These components build on x86_64, so add them to components list
-    components="$components flang-legacy pgmath flang flang_runtime"
     #components="$components hipfort"
     components="$components hipcc hipamd "
   fi
